### PR TITLE
Updated pl-rPL (Polish (Poland)) translation

### PR DESCRIPTION
**Pull request by 9Cube-dpustula**
Dodanie polskich tłumaczeń.
Translater : Daniel Pustuła.
Commit made via Stringlate

### CODE REVIEW

**Comment by flocke** [comment]
There are still quiet a few strings still missing, as soon as you add them I will gladly merge this.

**Comment by flocke** [comment]
If, for some reason, Stringlate doesn't show you all strings you can head over to Crowdin and do the translation there: https://crowdin.com/project/andotp

**Comment by flocke** [comment]
Please don't translate `settings.xml`, those are internal values only the four `strings_` files need to be translated.

**Comment by 9Cube-dpustula** [comment]
Sory for all truble, Stringlate suggest me this file too. 
I'll edit my commits and adjust to your advice.

**Comment by flocke** [comment]
No worries, maybe I have forgotten to mark some strings in there as non-translatable.

**Comment by 9Cube-dpustula** [comment]
I revert my modification in setting.xml and squash other commits into one.

**Comment by flocke** [comment]
For some reason settings.xml is still shown in the pull request.
Just remove the whole settings.xml file from your values-pl-rPL folder.

**Comment by 9Cube-dpustula** [comment]
My bad, merging join this file again. This time it'll be ok.

File: app/src/main/res/values-pl-rPL/strings_settings.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <string name="settings_title_label_size">Etykieta rozmiaru czcionki</string>
+
+    <string name="settings_title_openpgp_provider">Wybierz dostawcę OpenPGP</string>
+    <string name="settings_title_openpgp_keyid">Wybierz klucz OpenPGP</string>
+    <string name="settings_title_openpgp_sign">Podpisz zaszyfrowane kopie</string>
+    <string name="settings_title_openpgp_verify">Sprawdź poprawność zaszyfrowanych kopii</string>
+
+    <!-- Descriptions -->
+    <string name="settings_desc_tap_to_reveal">Domyślnie ukrywaj tokeny OTP, wymaga kliknięcia aby odkryć token</string>
+
```
Comment (flocke) [inline app/src/main/res/values-pl-rPL/strings_settings.xml]: Seems like you missed a string here: `settings_desc_auth_device`

File: app/src/main/res/values-pl-rPL/strings_main.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <!-- Dialogs -->
+    <string name="dialog_title_auth">Autoryzuj</string>
+    <string name="dialog_title_manual_entry">Wprowadź szczegóły</string>
+    <string name="dialog_title_remove">Usuń</string>
+    <string name="dialog_title_rename">Zmień nazwę</string>
+    <string name="dialog_title_security_backup">Bezpieczeństwo i kopie</string>
+
+    <string name="dialog_msg_auth">Podaj swoje dane uwierzytelniające aby uruchomić aplikacje</string>
+    <string name="dialog_msg_confirm_delete">Czy napewno usunąć to konto?</string>
+
```
Comment (flocke) [inline app/src/main/res/values-pl-rPL/strings_main.xml]: Here are three strings missing:
- `dialog_msg_security_backup_desc`
- `dialog_msg_security_backup_3rd_party`
- `dialog_msg_security_backup_warning`

File: app/src/main/res/values-pl-rPL/strings_main.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <string name="hint_label">Etykieta</string>
+    <string name="hint_secret">Sekret</string>
+
+    <!-- Labels -->
+    <string name="label_tap_to_reveal">Naciśnij aby odsłonić</string>
+    <string name="label_type">Napisz</string>
+    <string name="label_secret">Sekret</string>
+    <string name="label_period">Okres</string>
+    <string name="label_label">Etykieta</string>
+
```
Comment (flocke) [inline app/src/main/res/values-pl-rPL/strings_main.xml]: I just added a new string here: `label_algorithm`

**Comment by flocke** [comment]
I just added a few comments to your changes, there are a few strings missing (not many).
Once you add them I will merge this for release v0.2.5

**Comment by flocke** [comment]
I'm planing a new release this weekend. Could you maybe look at the strings again and update the translation if you got the time?
Easiest would be if you made a Crowdin account and translate over there, but you can also use Stringlate again and send a pull request.

**Comment by 9Cube-dpustula** [comment]
Translated and created account in Crowdin

**Comment by flocke** [comment]
Nice, I uploaded your translations from the pull requests to Crowdin so you only have to translate the new strings next time.

**Pull request merged**